packages/react-hooks/src/index.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { createContext, useState, useEffect, useContext } from 'react'
+
+import {
+  Agent,
+  ConnectionState,
+  CredentialState,
+  ProofState,
+  ConnectionEventTypes,
+  CredentialEventTypes,
+  ProofEventTypes,
+  ConnectionStateChangedEvent,
+  CredentialStateChangedEvent,
+  ProofStateChangedEvent,
+  ConnectionRecord,
+  ProofRecord,
+  CredentialRecord,
+  BasicMessageRecord,
+  BasicMessageReceivedEvent,
+  BasicMessageEventTypes,
+} from '@aries-framework/core'
+
+const TAG = 'Aries-Hooks:'
+
+const AgentContext = createContext<any>({})
+const ConnectionContext = createContext<any>({})
+const CredentialContext = createContext<any>({})
+const ProofContext = createContext<any>({})
+const BasicMessageContext = createContext<any>({})
+
+// Agent
+export const useAgent = (): { agent: Agent; loading: boolean } => {
+  return useContext(AgentContext)
+}
+
+// Connection
+export const useConnections = (): { connections: ConnectionRecord[]; loading: boolean } => {
+  return useContext(ConnectionContext)
+}
+
+export const useConnectionById = (id: string): ConnectionRecord | undefined => {
+  const { connections } = useContext(ConnectionContext)
+  const connection = connections.find((c: ConnectionRecord) => c.id === id)
+  if (!connection) {
+    console.warn(TAG, `Could not find Connection By ID ${id}`)
```

Comment:
please remove logging or use the agent logger

Reply:
I think this does provide some valuable info--not sure if it necessarily belongs in the AFJ logger, since this is an extension? Not super opinionated here though. 
On a second note, can we actually use the logger outside of AFJ? I wasn't aware that was being exposed properly (or maybe I'm being dumb 😆 )